setup.cfg

```diff
@@ -0,0 +1,27 @@
+[metadata]
+name = git-filter-repo
+url = https://github.com/newren/git-filter-repo
+project_urls = 
+    Source = https://github.com/newren/git-filter-repo
+    Issues = https://github.com/newren/git-filter-repo/issues/
+description = git filter-branch replacement
+version = 2.25.0
+long_description = file: README.md
+author = Elijah Newren
+classifiers =
+    Development Status :: 3 - Alpha
```

Comment:
In my experience, alpha/beta/production are developer-declared fuzzy this-is-what-i-feel-it-is declarations; does PyPI have more rigid requirements on these terms?  If it doesn't, I disagree with Alpha (at least on Linux/MacOSX; on windows Alpha may be fair.)

Reply:
It's just fuzzy what I feel yeah :)

Alpha is just what my packaging script emits by default, but you can choose whatever you "what-I-feel" -- what would you like? Mature?

Reply:
Yeah, that's a tougher call.  If you look at it from the known issues angle (Alpha software has lots of known issues, Beta has some, for Mature it'd be rare), then it'd be on the Mature side as I've squashed the known issues.  If you look at it from the well-tested angle, then although I've done lots of testing and have line coverage and know many people who have used it including some in production and including it in a library and passing along to their users, I still know there might be a tidal wave of in-the-wild testing to come since git-2.24.0 was recently released with pointers to filter-repo.  Who knows what all that real-life testing will turn up.

So, I dunno...I guess Beta for now?